openpype/hosts/unreal/hooks/pre_workfile_preparation.py

```diff
@@ -25,13 +27,45 @@ def __init__(self, *args, **kwargs):
 
         self.signature = "( {} )".format(self.__class__.__name__)
 
+    def _get_work_filename(self):
+        # Use last workfile if was found
+        last_workfile = self.data.get("last_workfile_path")
+        if last_workfile and os.path.exists(last_workfile):
+            return os.path.basename(last_workfile)
```

Comment:
```suggestion
        last_workfile = Path(self.data.get("last_workfile_path"))
        if last_workfile and last_workfile.exists():
            return last_workfile.name
```

Reply:
call me *pathlib*  nazi, but I would use only one way of handling path throughout the script - and because this is alway running in Python 3, let's use nice, clean, perfect pathlib api 😁

Reply:
In theory `self.data.get("last_workfile_path")` can be `None` is it Ok for `Path`?